datasets/wegenbestand/dataset.json

```diff
@@ -7,7 +7,7 @@
     "verkeer"
   ],
   "homepage": "https://data.amsterdam.nl",
-  "owner": "Gemeente Amsterdam, bereikbaarheidsthermometer Amsterdam",
+  "owner": "Gemeente Amsterdam, Afdeling Kennis en Kaders, Team Onderzoek en Kennis"",
```

Comment:
This doesn't look right: are there two quotes near the end of the line? Or is the comma missing? (I think it is the latter, but the _diff_ view is confusing.)